modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/atomic/GridDhtAtomicAbstractUpdateRequest.java

```diff
@@ -341,10 +393,24 @@ public final GridCacheVersion writeVersion() {
     }
 
     /**
-     * @return Cache write synchronization mode.
+     * @param writeVer New write version.
+     */
+    public void writeVersion(GridCacheVersion writeVer) {
+        this.writeVer = writeVer;
+    }
+
+    /**
+     * @return The write mode synchronization wrapper message.
+     */
+    public final CacheWriteSynchronizationModeMessage writeSynchronizationModeMessage() {
+        return syncModeMsg;
+    }
+
+    /**
+     * @param writeSyncModeMsg Write syncronization mode wrapper message.
      */
-    public final CacheWriteSynchronizationMode writeSynchronizationMode() {
```

Comment:
I don't see any usage of this method. Do we need to keep information about sync mode at all?

Reply:
double checked the logic. it seems we don't need mode information any longer